.github/workflows/main-branch.yaml

```diff
@@ -0,0 +1,56 @@
+name: "Main Branch: Build & Deploy Lambda"
+
+on:
+  push:
+    branches:
+    - main
+#    paths:
+#    - hello-world-python/**
+#    - .github/workflows/lambda-hello-world-python.yaml
+
+jobs:
+  config:
+    runs-on: self-hosted
```

Comment:
we've swapped to different self-hosted runner labels internally. We'll very likely need to install additional requirements to work on the new runner set (and so will customers once they start deploying the new component)

Reply:
`self-hosted-default` for example. Fully list here: [internal slack link](https://cloudposse.slack.com/archives/CA4TC65HS/p1702441032321329)